src/main/features/claim/views/confirmation.njk

```diff
@@ -15,8 +15,11 @@
           <span>{{ t('Your claim number:') }}</span>
           <h1 class="bold-large">{{ claimNumber }}</h1>
         </div>
+        <p>
+          <span class="bold">{{ claim.respondedAt | date }}</span>
```

Comment:
@cidlockie this is the date that the defendant responded to the claim not the one you want and is throwing an error because its not set yet, 
Is this supposed to be the issuedOn date which could be a future date if they've done it on a weekend / after 4pm etc, or the createdOn date which will be when they did it?